worker/src/Channel/ChannelSocket.cpp

```diff
@@ -191,6 +191,43 @@ namespace Channel
 		auto free = this->channelReadFn(
 		  &message, &messageLen, &messageCtx, this->uvReadHandle, this->channelReadCtx);
 
+		if (free)
```

Comment:
Please add some inline doc here telling what it does.

Reply:
Also, is this code executed when using Node? If so, could it be disabled via some define passed when building the worker?

Reply:
`ChannelSocket::CallbackRead` is not called by Node, this is just for direct communication with worker when we can make a function call. Will add some comments.

Reply:
Do added comments make sense?

Reply:
Yes, but I miss some comments telling that the block is only used in Rust due to how message channel works in Rust (within some process) compared to how it works in Node (inter-process comm).

Reply:
Hm, if you go to the place from which this function is called, it makes total sense

Reply:
I described the whole workflow in https://github.com/versatica/mediasoup/pull/666 that introduced this code

Reply:
Merge please 😀